src/docs/perf/rendering/shader.md

```diff
@@ -214,14 +201,35 @@ difference as illustrated in the beginning of this article.
    along these lines if this is something that interests you. To get started, please
    see our [contribution guidelines].
 
+4. **This process would be easier if the `flutter` tool could do X!**<br><br>
+   There are a number of possible ways that the tooling around shader warm-up
+   could be improved. Please let us know what is important to you by giving
+   a thumbs-up to your feature request on GitHub, or by filing a new one if
```

Comment:
Do we have a good link for this? For example, I tried this one:  https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+shader+jank+label%3A%22severe%3A+rendering%22

But it would be a lot more useful if we could provide a link to open "shader warmup/jank" issues.

Reply:
Add a link.